src/Compute/Compute.Autorest/docs/Remove-AzGalleryApplication.md

```diff
@@ -55,7 +55,8 @@ Accept wildcard characters: False
 ```
 
 ### -DefaultProfile
-The credentials, account, tenant, and subscription used for communication with Azure.
+The DefaultProfile parameter is not functional.
```

Comment:
Adam made a comment on these in chat, but where are these gallery related cmdlet changes coming from? 
Can we leave them as they are ? Was it suggested by the PowerShell team?